Jetcaster/wear/src/main/java/com/example/jetcaster/JetcasterApplication.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.jetcaster
+
+import android.app.Application
+import android.os.StrictMode
+import coil.ImageLoader
+import coil.ImageLoaderFactory
+
+class JetcasterApplication : Application(), ImageLoaderFactory {
```

Comment:
Consider renaming to `JetcasterWearApplication` to differentiate this from the mobile version. (I should probably rename the mobile one to `JetcasterMobileApplication`

Reply:
done